docs: added shebang information (#1796)
nodejs/nodejs.dev: src/documentation/0007-node-run-cli/index.md

node app.js
```

> Above you are explicitly telling the shell how to run your script. You can also embed this information into JavaScript file through shebang line. Shebang is the first line in the file which tells OS which interpreter to use for running the script, below is the first line of JavaScript:

Reviewer: ```suggestion
Above, you are explicitly telling the shell to run your script with `node`. You can also embed this information into your JavaScript file with a "shebang" line. The "shebang" is the first line in the file, and tells the OS which interpreter to use for running the script. Below is the first line of JavaScript:
```

```bash
#!/usr/bin/node
```

> Above we are explicitly giving the absolute path of interpreter but not all OS have `node` in it's bin folder but all have `env` so you can tell OS to run env with node as parameter:

Reviewer: ```suggestion
Above, we are explicitly giving the absolute path of interpreter. Not all operating systems have `node` in the bin folder, but all should have `env`. You can tell the OS to run `env` with node as parameter:
```

```bash
#!/usr/bin/env node

// your code
```

> For using shebang your file should have executable permission which you can give as follows:

Reviewer: ```suggestion
To use a shebang, your file should have executable permission. You can give `app.js` the executable permission by running:
```

node app.js
```

> Above you are explicitly telling the shell how to run your script. You can also embed this information into javascript file through shebang line. Shebang is the first line in the file which tells os which interpreter to use for running the script, below is the first line of javascript

Reviewer 2: ```suggestion
Above you are explicitly telling the shell how to run your script. You can also embed this information into JavaScript file through shebang line. Shebang is the first line in the file which tells OS which interpreter to use for running the script, below is the first line of JavaScript:
```

```javascript
#!/usr/bin/node

// your code
```

> Above we are explicitly giving the absolute path of interpreter but not all os have `node` in it's bin folder but all have `env` so you can tell os to run env with node as parameter

Reviewer 2: ```suggestion
Above we are explicitly giving the absolute path of interpreter but not all OS have `node` in it's bin folder but all have `env` so you can tell OS to run env with node as parameter:
```

```javascript
#!/usr/bin/env node

// your code
```

> For using shebang your file should have executable permission which you can give as follows

Reviewer 2: ```suggestion
For using shebang your file should have executable permission which you can give as follows:
```
